src/Directory.Build.targets

```diff
@@ -13,10 +13,21 @@
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);IncludePDBsInPackage</TargetsForTfmSpecificContentInPackage>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Include="$(ProjectDir)\Scripts\*.targets">     
+    <PackagePath>build</PackagePath>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="$(ProjectDir)\Scripts\*.ps1">     
+    <PackagePath>build/Scripts</PackagePath>
```

Comment:
@janovesk - Should the path be "Deployment/Msmq"  instead? ping @andreasohlund

Reply:
You probably mean line 5 in **NServiceBus.Transport.Msmq.targets** instead. This line only says where to put the scripts in the nuget. They should go under **build** to avoid Nuget from copying the scripts into the project referencing it.